lib/components/BuilderForm.jsx

```diff
@@ -1,223 +1,289 @@
+                  label={
+                    <>
+                      Sponsorship Symbol: <span className='text-default italic'>(eg. 'SPON')</span>
+                    </>
+                  }
+                  placeholder='(eg. DSPON)'
+                  required
+                  onChange={(e) => {
+                    setUserChangedSponsorshipTicker(true)
+                    setSponsorshipSymbol(e.target.value)
+                  }}
+                  value={sponsorshipSymbol}
+                />
+              </InputLabel>
 
-        <TextInputGroup
-          id='_ticketSymbol'
-          label={
-            <>
-              Ticket Symbol:{' '}
-              <span className='text-default italic'>(eg. 'TICK')</span>
-            </>
-          }
-          placeholder='(eg. DTICK)'
-          required
-          onChange={e => setTicketSymbol(e.target.value)}
-          value={ticketSymbol}
-        />
+              <InputLabel 
+                title='Credit Limit'
+                description='A percentage, eg. 10 is 10%. Defaults to 10%.'
+                className="mb-8"
+              >
+                <TextInputGroup
+                  id='_ticketCreditLimitPercentage'
+                  required
+                  type='number'
+                  pattern='\d+'
+                  onChange={(e) => {
+                    if (!userChangedMaxExitFee) {
+                      setMaxExitFeePercentage(Math.round(e.target.value * 4))
+                    }
+                    setTicketCreditLimitPercentage(e.target.value)
+                  }}
+                  value={ticketCreditLimitPercentage}
+                />
+              </InputLabel>
 
-        <TextInputGroup
-          id='_maxExitFeeMantissa'
-          label={
-            <>
-              Max Exit Fee:{' '}
-              <span className='text-default italic'>
-                (a percentage in decimals, eg. 0.5 == 50%)
-              </span>
-            </>
-          }
-          required
-          onChange={e => setMaxExitFeeMantissa(e.target.value)}
-          value={maxExitFeeMantissa}
-        />
+              <InputLabel 
+                title='Credit Maturation'
+                description='Duration in days. Defaults to 1x the Prize Period.'
+                className="mb-8"
+              >
+                <TextInputGroup
+                  id='_creditMaturationInDays'
+                  required
+                  type='number'
+                  pattern='\d+'
+                  onChange={(e) => {
+                    setUserChangedCreditMaturation(true)
+                    setCreditMaturationInDays(e.target.value)
+                  }}
+                  value={creditMaturationInDays}
+                />
+              </InputLabel>
+              
+              <InputLabel 
+                title='Max Exit Fee'
+                description='A percentage, eg. 10 is 10%. Defaults to 4x the Credit Limit.'
+                className="mb-8"
```

Comment:
I didn't mention this but I'd love to stick to using single quotes for every string, unless it needs interpolation (ie. backticks ``)